src/main/java/bossmonster/view/OutputProcessorBySystemOut.java

```diff
@@ -0,0 +1,13 @@
+package bossmonster.view;
+
+public class OutputProcessorBySystemOut implements OutputProcessor {
+    @Override
+    public void print(Object result) {
+        System.out.println(result);
+    }
+
+    @Override
+    public void printError(Exception e) {
+        System.out.println(String.format("[ERROR] %s",e.getMessage()));
```

Comment:
```suggestion
        System.out.println(String.format("[ERROR] %s", e.getMessage()));
```
`String.format()` 사용 좋습니다👍👍  다만 띄어쓰기가 있었으면 좋겠어요

Reply:
이런 컨벤션을 지켜본 적이 없어서 또 실수했네요.. 매번 확인해보고, ide 설정을 바꿔봐야 할 것 같습니다.